src/ray/core_worker/lib/java/org_ray_runtime_RayNativeRuntime.cc

```diff
@@ -108,12 +108,15 @@ JNIEXPORT void JNICALL Java_org_ray_runtime_RayNativeRuntime_nativeSetup(JNIEnv
                                                                          jstring logDir) {
   std::string log_dir = JavaStringToNativeString(env, logDir);
   ray::RayLog::StartRayLog("java_worker", ray::RayLogLevel::INFO, log_dir);
-  // TODO (kfstorm): If we add InstallFailureSignalHandler here, Java test may crash.
+  // TODO (kfstorm): We can't InstallFailureSignalHandler here, because JVM already
+  // installed its own signale handler.
```

Comment:
good to also mention "It's possible to fix this by chaining signal handlers but it's not easy, for more details, see ... "

Reply:
Done.